api/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from PIL import Image
+from io import BytesIO
+
+
+def make_request(url, params=None):
+    response = requests.get(url, params=params)
+    return response.json()
```

Comment:
Is it used?

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/aa95766f824ad79eea1e616ef7c7ca1b03b6f8ea